dgraph/cmd/alpha/run.go

```diff
@@ -108,11 +108,12 @@ they form a Raft group and provide synchronous replication.
 	flag.StringP("postings", "p", "p", "Directory to store posting lists.")
 	flag.String("tmp", "t", "Directory to store temporary buffers.")
 
-	// Options around how to set up Badger.
-	flag.String("badger.compression", "snappy",
-		"[none, zstd:level, snappy] Specifies the compression algorithm and the compression"+
-			"level (if applicable) for the postings directory. none would disable compression,"+
-			" while zstd:1 would set zstd compression at level 1.")
+	flag.String("badger", worker.BadgerDefaults,
+		`Various badger options.
+	goroutines=N provides the number of goroutines to use in badger.Stream.
```

Comment:
We need this `goroutines` flag in Alpha for [creating local threads during backups](https://github.com/dgraph-io/dgraph/pull/7387/files#diff-e64b7a1608a78718294f5229733bcdfc7a383ecb5a359dd82ab35883874a94cbR63), and Badger uses the `goroutines` value as the default `Stream.numGo`.